src/Avalonia.Controls/Button.cs

```diff
@@ -667,7 +667,7 @@ private void UpdatePseudoClasses()
 
         void ICommandSource.CanExecuteChanged(object sender, EventArgs e) => this.CanExecuteChanged(sender, e);
 
-        void IClickableControl.RaiseClick() => OnClick();
+        void IClickableControl.RaiseClick() => OnClick(null);
```

Comment:
In response to your question in the PR description: I'd say this is correct to pass `null` when the event is raised programatically.